ui/app/styles/core/buttons.scss

```diff
@@ -91,12 +91,6 @@
     color: $red-500;
   }
 
-  &.is-warning-outlined {
```

Comment:
Removing because this color doesn't exist in [HDS buttons](https://helios.hashicorp.design/components/button?tab=code#component-api)